library_schemas/score/prerelease/HED_score_1.1.1.mediawiki

```diff
@@ -12,953 +12,587 @@ For more information see https://hed-schema-library.readthedocs.io/en/latest/ind
         ** EMG-muscle-name
             *** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-        ** EMG-feature
-            *** EMG-myoclonus
-                **** Negative-myoclonus
-                    ***** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-                **** EMG-myoclonus-rhythmic
-                        ***** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-                    **** EMG-myoclonus-arrhythmic
-                        ***** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-                    **** EMG-myoclonus-synchronous
-                        ***** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-                    **** EMG-myoclonus-asynchronous
-                        ***** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-            *** EMG-PLMS <nowiki>[Periodic limb movements in sleep.]</nowiki>
-            *** EMG-spasm
-                **** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-            *** EMG-tonic-contraction
-                **** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-            *** EMG-asymmetric-activation 
-              **** EMG-asymmetric-activation-left-first
-                  ***** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-              **** EMG-asymmetric-activation-right-first
-                  ***** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-            *** Other-EMG-features {requireChild}
-                **** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-    * Other-polygraphic-channel {requireChild}
+        ** Myoclonus <nowiki> [Source: Beniczky ea 2017, Table 16.] </nowiki>
+            *** Negative-myoclonus <nowiki> [Source: Beniczky ea 2017, Table 16.] </nowiki>
+            *** Myoclonus-rhythmic <nowiki> [Numerical value for frequency. (Source: Beniczky ea 2017, Table 16.)] </nowiki>
+                *** <nowiki># {takesValue, valueClass=numericClass} </nowiki>
+            *** Myoclonus-arrhythmic <nowiki> [Source: Beniczky ea 2017, Table 16.] </nowiki>
+            *** Myoclonus-synchronous <nowiki> [Source: Beniczky ea 2017, Table 16.] </nowiki>
+            *** Myoclonus-asynchronous <nowiki> [Source: Beniczky ea 2017, Table 16.] </nowiki>
+        ** PLMS <nowiki>[Periodic limb movements in sleep. (Source: Beniczky ea 2017, Table 16.)]</nowiki>
+        ** Spasm <nowiki> [Source: Beniczky ea 2017, Table 16.] </nowiki>
+        ** Tonic-contraction <nowiki> [Source: Beniczky ea 2017, Table 16.] </nowiki>
```

Comment:
Description:
"A sustained muscle contraction evoked when the motor nerve that innervates a skeletal muscle emits action potentials at a very high rate. (Source: Wikipedia; Beniczky ea 2017, Table 16.)"